Code/GraphMol/MolStandardize/TautomerCatalog/TautomerCatalogUtils.h

```diff
@@ -35,7 +35,7 @@ class TautomerCatalogParams;
 // typedef std::vector<ROMol*, std::string, std::string> tautomerTransform;
 class RDKIT_MOLSTANDARDIZE_EXPORT TautomerTransform {
  public:
-  ROMol* Mol;
+  ROMol* Mol = nullptr;
```

Comment:
Could just make this a unique_ptr

Reply:
yeah, we could, but that's a public API change (not necessarily tragic here because *probably* no one is using that public data member)
Is the benefit of that important enough to change the API?